.test-infra/jenkins/job_PreCommit_Java_Examples_Dataflow_Java11.groovy

```diff
@@ -46,7 +46,6 @@ builder.build {
     gradle {
       rootBuildScriptDir(properties.checkoutDir)
       tasks 'javaExamplesDataflowPreCommit'
-      switches '-PimageJavaVersion=11'
```

Comment:
since this uses the runner buildAndPushDockerContainer task which now checks for -PcompileAndRunTestsWithJava11, we don't need this flag anymore